.github/workflows/build.yml

```diff
@@ -80,5 +80,15 @@ jobs:
             title: 'Update schema (auto-generated)',
             head: api_version + '-auto-generated',
             base: 'master',
-            body: "Release notes:\n - https://shopify.dev/concepts/about-apis/versioning/release-notes/" + api_version
+            body: ""Release notes:" + 
+              "\n- https://shopify.dev/concepts/about-apis/versioning/release-notes/" + api_version + "#storefront-api-changes" +
+              "\n" +
+              "\n**Steps to release:**" +
+              "\n- [ ] Make sure tests are passing" +
+              "\n- [ ] Navigate to the [Shopify Android bintray website](https://bintray.com/beta/#/shopify/shopify-android?tab=overview) and check that the latest version has been uploaded." +
```

Comment:
You are introducing a deploy workflow which gets triggered AFTER a Github release has been published, so why do you expect the deployment to have already happened. I would suggest removing this line unless there is something I'm missing.